.github/workflows/ci.yml

```diff
@@ -214,56 +213,4 @@ jobs:
         run: pnpm run build
 
       - name: Test
-        run: pnpm run test:smoke
-
```

Comment:
Deleted a few lines too high!

Reply:
not deleted! you can see it as the new line 216. Not sure why Git shows this as deleted though